cmd/snap-update-ns/change.go

```diff
@@ -121,6 +120,23 @@ func (c *Change) createPath(path string, pokeHoles bool, as *Assumptions) ([]*Ch
 		err = MkfileAll(path, mode, uid, gid, rs)
 	case "symlink":
 		err = MksymlinkAll(path, mode, uid, gid, c.Entry.XSnapdSymlink(), rs)
+	case "ensure-dir":
```

Comment:
This needs some unit tests.

Reply:
https://github.com/snapcore/snapd/pull/13342/files#r1404221657